lib/util/utils.dart

```diff
@@ -316,6 +316,7 @@ class Utils {
             onError: Utils.getAction(payload['onError']),
             uploadApi: payload['uploadApi'],
             inputs: inputs,
+            fieldName: Utils.optionalString(payload['fieldName']),
```

Comment:
@snehmehta other option is to have fieldName as non-nullable and default to 'files'. Then use Utils.getString with the fallback as the default value. That way it will never be null so no checks needed. Your call